src/main/java/core/basesyntax/Plane.java

```diff
@@ -1,11 +1,83 @@
 package core.basesyntax;
 
 /**
- * <p>Дано класс Plane. Написать класс PlaneBuilder, который реализует паттерн Builder.
+ * <p>Дано класс Plane. Написать класс PlaneBuilder, который реализует паттерн PlaneBuilder.
  * Добавьте не менее пяти полей в класс Plain. Внести все необходимые изминения в класс Plane
- * которые могут потребоваться для реализации паттерна Builder.</p>
+ * которые могут потребоваться для реализации паттерна PlaneBuilder.</p>
  */
 
 public class Plane {
+    private String planeType;
+    private int numberOfPassengers;
+    private String yearOfProduction;
+    private String whichAirlines;
+    private int code;
 
+    private Plane(PlaneBuilder planeBuilder) {
+    }
```

Comment:
your solution works but this constructor is useless.

Reply:
done